RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Cumulative Release Notes for the Annotated Data Model
 
+### 2.1.5
+
+### [ROS-254](https://basistech.atlassian.net/browse/ROS-254) Add  representation for dependencies.
```

Comment:
Your unit test is for "Dogs eat books."  I see "V", "S", O", etc. and two indexes for each.
But I (and other ADM users who haven't been here yet) don't know what these really
mean.  I'm just asking for a tiny example like this, with a bit of prose saying "the S mean
this token is the subject, the governor index point to "dog", etc...